drivers/ethernet/eth_stm32_hal.c

```diff
@@ -78,6 +78,13 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define CACHE
 #endif
 
+#if defined(CONFIG_SOC_SERIES_STM32H7X) && DT_NODE_HAS_STATUS(DT_NODELABEL(sram3), okay)
```

Comment:
Should there still be a Kconfig option to enable this for the driver?

Reply:
@erwango ^^